File: benchmarks/python/fibonacci.py
def fib(n):
    if n <= 1:
        return n
    else:
        return fib(n - 1) + fib(n - 2)

def benchmark_fib(n, iterations):
    for _ in range(iterations):
        fib(n)

benchmark_fib(30, 10)

File: benchmarks/ruby/fibonacci.rb
def fib(n)
  if n <= 1
    n
  else
    fib(n - 1) + fib(n - 2)
  end
end

def benchmark_fib(n, iterations)
  iterations.times do
    fib(n)
  end
end

benchmark_fib(30, 10)